src/App.jsx

```diff
@@ -1,26 +1,26 @@
-import React, { useEffect } from "react";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+// import React, { useEffect } from "react";
+// import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 
-import Login from "./login/login.jsx";
-import HomePage from "./HomePage";
-import Register from "./login/register.jsx";
-import { AuthProvider } from "./contexts/authContext";
+// import Login from "./login/login.jsx";
+// import HomePage from "./HomePage";
+// import Register from "./login/register.jsx";
+// import { AuthProvider } from "./contexts/authContext";
 
-function App() {
-  return (
-    <AuthProvider>
-      <Router>
-        <div>
-          <Routes>
-            <Route path="/" element={<Login />} />
-            <Route path="/register" element={<Register />} />
-            <Route path="/Home" element={<HomePage />} />
-            {/* <Route component={NotFound} /> */}
-          </Routes>
-        </div>
-      </Router>
-    </AuthProvider>
-  );
-}
+// function App() {
+//   return (
+//     <AuthProvider>
+//       <Router>
+//         <div>
+//           <Routes>
+//             <Route path="/" element={<Login />} />
+//             <Route path="/register" element={<Register />} />
```

Comment:
@Capn05 the Register button loads an empty page. Can check if its still working for you and provide any steps if additional setup is required? I'm assuming not, since all the firebase login info is hardcoded

Reply:
It should be working now